src/analyzer/Analyzer.cc

```diff
@@ -116,8 +116,8 @@ Analyzer::~Analyzer() {
     assert(finished);
     assert(new_children.empty());
 
-    LOOP_OVER_CHILDREN(i)
```

Comment:
Oh, thank you!

I see you kept the `LOOP_OVER_CHILDREN` macros in Analyzer.h.

Can we remove them, or are you worried they are part of the API? If they are part of the API, maybe we could change the expansion such that a deprecation warning is produced?

Maybe something like this? https://stackoverflow.com/a/2685696

Reply:
I'm worried they're part of the API. I'll take a look at deprecating them though. Probably part of a separate PR though.